linear_operator/test/linear_operator_test_case.py

```diff
@@ -617,7 +625,11 @@ def test_add_low_rank(self):
         new_rows = torch.randn(*linear_op.shape[:-1], 3)
 
         summed_lt = evaluated + new_rows.matmul(new_rows.mT)
-        new_lt = linear_op.add_low_rank(new_rows)
+        try:
+            new_lt = linear_op.add_low_rank(new_rows)
+        except Exception:
+            msg = traceback.format_exc()
```

Comment:
What is the purpose of this?

Reply:
I was having trouble pinning down a function signature error and temporarily added exception details. Meant to remove this. Reverted.